### PR TITLE
New deploy option: build-only

### PR DESCRIPTION
**Pull request by filipenevola**
Some clients want to have the ability to build first and after some validations proceeding with the upload and deploy.

### CODE REVIEW

File: tools/meteor-services/deploy.js [hunk 1]
```diff
@@ -481,45 +481,59 @@ export async function bundleAndDeploy(options) {
...
-  // or is revoked, user comes back to deploy again. In that case,
-  // they'll get an email prompt instead of a username prompt because
-  // the command-line tool didn't have time to learn about their
-  // username before the credential was expired.
-  pollForRegistrationCompletion({
-    noLogout: true
-  });
-  var promptIfAuthFails = (loggedInUsername() !== null);
+  if (options.isBuildOnly) {
+    Console.info('Skipping pre authentication as the option --build-only was informed.');
```
Comment (StorytellerCZ) [inline tools/meteor-services/deploy.js:489]: `was informed` shouldn't this be `was provided`?

File: tools/meteor-services/deploy.js [hunk 2]
```diff
@@ -609,6 +623,13 @@ export async function bundleAndDeploy(options) {
     });
   }
 
+  if (options.isBuildOnly) {
+    Console.info(
```
Comment (StorytellerCZ) [inline tools/meteor-services/deploy.js:627]: I think it would be better to use `Console.success` here to make it clear that the process has finished.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

File: tools/meteor-services/deploy.js (hunk 2 above)
Comment (filipenevola) [reply tools/meteor-services/deploy.js:625]: As we are not using success anywhere in the deploy maybe it's better to just keep as info.

Comment (StorytellerCZ) [reply tools/meteor-services/deploy.js:625]: We are using error, but let's shelve this for a bigger overview then.

**Review by StorytellerCZ — APPROVED** [review]

**Pull request merged**